apps/trading-e2e/src/integration/trading-portfolio.cy.ts

```diff
@@ -55,7 +55,7 @@ describe('Portfolio page', { tags: '@smoke' }, () => {
       cy.get('fieldset.ag-simple-filter-body-wrapper')
         .should('be.visible')
         .within((fields) => {
-          cy.wrap(fields).find('label').should('have.length', 16);
```

Comment:
**nitpick (non-blocking)**: Ultra-nitpick, but I can't see *why* this test had to change. But it's a label so... nitpick.

Reply:
updated fields in portfolio after re-generating types